src/api/c++/z3++.h

```diff
@@ -593,23 +675,18 @@ namespace z3 {
     /**
        \brief A Z3 sort (aka type). Every expression (i.e., formula or term) in Z3 has a sort.
     */
-    class sort : public ast {
+    class sort final : public ast {
     public:
         sort(context & c):ast(c) {}
         sort(context & c, Z3_sort s):ast(c, reinterpret_cast<Z3_ast>(s)) {}
         sort(context & c, Z3_ast a):ast(c, a) {}
-        sort(sort const & s):ast(s) {}
```

Comment:
This, and the operator below, constitute a rule of 5 violation when alone. Since they are both 'default' (i.e. do nothing differently than if they were implicitly generated), we just delete them. The compiler will implicitly create these methods and since this fixes the rule of 5 violation, the other impilict methods will be safe to use.